index.html

```diff
@@ -3225,36 +3235,37 @@ <h3>Terms of Use</h3>
     "https://www.w3.org/ns/credentials/v2",
     "https://www.w3.org/ns/credentials/examples/v2"
   ],
-  "id": "http://university.example/credentials/3732",
-  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
-  "issuer": "https://university.example/issuers/14",
-  "validFrom": "2010-01-01T19:23:24Z",
-  "credentialSubject": {
-    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "degree": {
-      "type": "ExampleBachelorDegree",
-      "name": "Bachelor of Science and Arts"
-    }
+  id: "urn:did:123456",
+  type: ["VerifiableCredential", "VerifiableAttestation", "VerifiableId"],
+  issuer: "did:ebsi:zz7XsC9ixAXuZecoD9sZEM1",
+  issuanceDate: "2021-11-01T00:00:00Z",
+  validFrom: "2021-11-01T00:00:00Z",
+  issued: "2021-10-30T00:00:00Z",
+  credentialSubject: {
+    id: "did:key:z2dmzD81cgPx8Vki7JbuuMmFYrWPgYoytykUZ3eyqht1j9KbrDt4zxXoDrBWYFiATYZ8G9JMeEXC7Kki24fbTwtsJbGe5qcbkYFunSzcDokMRmj8UJ1PbdCGh33mf97K3To89bMzd15qrYq3VkDztoZqfmujkJVpvTbqoXWXqxmzNDbvMJ",
+    personalIdentifier: "IT/DE/1234",
+    familyName: "Castafiori",
+    firstName: "Bianca",
+    dateOfBirth: "1930-10-01",
   },
-  <span class="highlight">"termsOfUse": [{
-    "type": "IssuerPolicy",
-    "id": "http://example.com/policies/credential/4",
-    "profile": "http://example.com/profiles/credential",
-    "prohibition": [{
-      "assigner": "https://university.example/issuers/14",
-      "assignee": "AllVerifiers",
-      "target": "http://university.example/credentials/3732",
-      "action": ["Archival"]
-    }]
-  }]
-  </span>
+  credentialSchema: {
+    id: "https://api-test.ebsi.eu/trusted-schemas-registry/v2/schemas/z3MgUFUkb722uq4x3dv5yAJmnNmzDFeK5UC8x83QoeLJM",
+    type: "FullJsonSchemaValidator2021",
+  },
+  expirationDate: "2031-11-30T00:00:00Z",
+  termsOfUse: {
+    id: "https://api-test.ebsi.eu/trusted-issuers-registry/v4/issuers/did:ebsi:zz7XsC9ixAXuZecoD9sZEM1/attributes/7201d95fef05f72667f5454c2192da2aa30d9e052eeddea7651b47718d6f31b0",
```

Comment:
```suggestion
    id: "https://vendor.example/trusted-issuers-registry/v4/issuers/did:example:123/attributes/456",
```

Reply:
Not done, because the id URL actually works and can be resolved. It is a real example as asked for.

Reply:
do you have permission to paint them as a target like this : )

Using a real endpoint like this could cost them money, automated crawlers will hit it, etc...

IMO its better to use W3C to host things, or to include the dereferenced content as a separate example

Reply:
Let's discuss this at a WG meeting. If Ivan can host the URL then we can copy the contents to there.

However, at a higher level, there is a conflict that the WG needs to resolve, between asking for a real life example and then complaining that the real life example contains a real life URL.

Reply:
> [@David-Chadwick] Let's discuss this at a WG meeting

Has this been discussed and resolved? If not, I suggest creating an issue for it.